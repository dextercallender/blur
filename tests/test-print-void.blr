void retVoid() {

}

void main() {
    print(retVoid());
}
