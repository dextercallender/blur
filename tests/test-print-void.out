Fatal error: exception Failure("Cannot print void.")